src/clr-angular/utils/popover/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ *
+ */
+
+export * from './enums/alignment.enum';
+export * from './enums/axis.enum';
+export * from './enums/side.enum';
+export * from './interfaces/popover-position.interface';
+export * from './providers/popover-events.service';
```

Comment:
Any services that aren't meant to be consumed by users of Clarity shouldn't be in the index, looks like these are not prefixed with Clr so they won't be public right? If so, remove these or add the Clr prefix.

Reply:
These need to be public.I wIll add the `Clr` prefix back onto them.